### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by Aksana-Tsishchanka**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -12,6 +12,8 @@
   ],
   "globals": {
     "console": true,
-    "module": true
+    "module": true,
```
Comment (boccob) [inline .eslintrc.json]: You can specify environment instead:
https://eslint.org/docs/user-guide/configuring#specifying-environments

File: dirwatcher/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import EventEmitter from 'events';
+import { default as Chokidar } from 'chokidar';
```
Comment (boccob) [inline dirwatcher/index.js]: `import Chokidar from 'chokidar';` :)

File: importer/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  static async import(filePath) {
+    return await csvJsoAsync(filePath);
+  }
+  
+  static importSync(filePath) {
+    return csvJsonSync(filePath);
+  }
+}
+
+function csvJsoAsync(path) {
```
Comment (boccob) [inline importer/index.js]: Typo

File: importer/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  return new Promise((resolve, reject) => {
```
Comment (boccob) [inline importer/index.js]: You can use `util.promisify` from node 8, 'pify' or 'es6-promisify' and use async / await with `fs.readFile`.

File: importer/index.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    let options = {
+      delimiter : ',',
+      quote     : '"'
+    };
+    
+    fs.readFile(path, { encoding : 'utf8' }, (error, data) => {
+      if (error) reject(error)
```
Comment (boccob) [inline importer/index.js]: Awesome code style

File: package.json [hunk 6]
```diff
@@ -4,8 +4,10 @@
   "description": "task1 for Node.js mentoring program",
   "main": "index.js",
   "scripts": {
-    "start": "babel-node index.js",
-    "test": "node"
+    "compile": "babel-node index.js",
+    "test": "node",
+    "start": "nodemon --exec npm run compile",
+    "build": "mkdir -p build && babel index.js --out-file build/index.js"
```
Comment (boccob) [inline package.json]: Nice idea, but it does not work because you transpile only one file.

File: package.json [hunk 7]
```diff
@@ -21,11 +23,16 @@
   },
   "homepage": "https://github.com/Aksana-Tsishchanka/npm_task1#readme",
   "devDependencies": {
-    "eslint": "4.6.1"
+    "eslint": "4.6.1",
```
Comment (boccob) [inline package.json]: Don't know why but it does not work for me. I get `Error: Cannot find module 'escope'`.

File: package.json [hunk 8]
```diff
@@ -21,11 +23,16 @@
...
   },
   "dependencies": {
     "babel-cli": "6.26.0",
     "babel-preset-env": "1.6.0",
-    "babel-preset-stage-2": "6.24.1"
+    "babel-preset-stage-2": "6.24.1",
+    "chokidar": "^1.7.0",
+    "csvjson": "^4.3.4",
+    "csvtojson": "^1.1.7",
+    "deasync": "^0.1.10"
```
Comment (boccob) [inline package.json]: Not used

Comment (Aksana-Tsishchanka) [reply package.json]: thx

File: package.json (hunk 6 above)
Comment (Aksana-Tsishchanka) [reply package.json]: I've moved a source code to folder 'src' and now the whole folder is compiled, thx for the note!

File: .eslintrc.json (hunk 1 above)
Comment (Aksana-Tsishchanka) [reply .eslintrc.json]: thx, added

**Review by boccob — APPROVED** [review]

**Pull request merged**